### PR TITLE
E2E: Parameterize Density failure

### PR DESCRIPTION
**Pull request by jayunit100**

### CODE REVIEW

**Comment by mbforbes** [comment]
Assigning to @lavalamp for context in density tests; +cc @wojtek-t @rrati @fgrzadkowski

File: test/e2e/util.go [hunk 1]
```diff
@@ -444,11 +444,14 @@ func DeleteRC(c *client.Client, ns, name string) error {
 	return nil
 }
 
-// Launch a Replication Controller and wait for all pods it spawns
-// to become running. The controller will need to be cleaned up external
-// to this method
+// RunRC Launches (and verifies correctness) of a Replication Controller
+// It will waits for all pods it spawns to become "Running".
+// The controller will need to be cleaned up external to this method.
```
Comment (lavalamp) [inline test/e2e/util.go]: nit: Last line would read better as "It's the caller's responsibility to clean up the controller."

File: test/e2e/util.go [hunk 2]
```diff
@@ -444,11 +444,14 @@ func DeleteRC(c *client.Client, ns, name string) error {
...
 func RunRC(c *client.Client, name string, ns, image string, replicas int) error {
 	var last int
+
+	//TODO: We might want to make "maxContainerFailures" a factor of the replicas.
```
Comment (lavalamp) [inline test/e2e/util.go]: Comments should have a space after the '//'-- did this get gofmt'd?

Comment (lavalamp) [inline test/e2e/util.go]: I suggest just making it something in the range of .5% - 5% of replicas, minimum 1. That seems clearly better than a straight 10.

**Comment by lavalamp** [comment]
LGTM, just nits

File: test/e2e/util.go (hunk 2 above)
Comment (jayunit100) [inline test/e2e/util.go]: I like the percentage idea as well.

**Comment by jayunit100** [comment]
okay fixed ! , pushing now....

**Comment by lavalamp** [comment]
LGTM

**Comment by lavalamp** [comment]
Shippable failure is unrelated

**Comment by wojtek-t** [comment]
LGTM - thanks for this change!

**Pull request merged**